### PR TITLE
Welcome page

### PR DESCRIPTION
**Pull request by jdmcleod**
## Why?

Users will now be able to see a welcome page and navigate back and forth between it and the calculator

I'm not happy at all with how the welcome page looks. I'm really bad with CSS just because I haven't used it a whole ton.

## What Changed

Please list the changes in this PR?

* [ ] Added navbar
* [ ] Added basic welcome page
* [ ] Change routes

### CODE REVIEW

File: views/stylesheets/application.scss [hunk 1]
```diff
@@ -2,12 +2,13 @@
 
 $black: black;
 $white: white;
-$primary: #4793e3;
+$primary: #4792e39c;
```
Comment (Jeremy-Walton) [inline views/stylesheets/application.scss]: Did you mean to change this? This isn't a valid hex value.

**Review by Jeremy-Walton — CHANGES_REQUESTED** [review]
Looks good other than the one thing

File: views/stylesheets/application.scss (hunk 1 above)
Comment (jdmcleod) [reply views/stylesheets/application.scss]: Yes, it changes to that when I bump down the opacity.

File: views/stylesheets/application.scss [hunk 2]
```diff
@@ -2,12 +2,13 @@
 
 $black: black;
 $white: white;
-$primary: #4793e3;
+$primary: #4792e3;
```
Comment (Jeremy-Walton) [inline views/stylesheets/application.scss:5]: Why did this change?

File: views/index.slim [hunk 3]
```diff
@@ -36,13 +36,34 @@
                       option 2
                       option 3
               .row.col-md-3.justify-content-center
-                button.btn.btn-success type="submit" Proccess Roll
+                button.btn.btn-success type="submit" disabled="true" id="process" Process Roll
+                javascript:
+                  function enableButton()
+                  {
```
Comment (Jeremy-Walton) [inline views/index.slim:42]: We will need to clean this up later but for now, it's fine.

**Review by Jeremy-Walton — DISMISSED** [review]

File: views/stylesheets/application.scss (hunk 2 above)
Comment (jdmcleod) [reply views/stylesheets/application.scss:5]: I just changed the background color a bit.

Comment (jdmcleod) [reply views/stylesheets/application.scss:5]: I changed the background color slightly.

**Review by Jeremy-Walton — APPROVED** [review]

**Pull request merged**